src/handlers/messageHandler.ts

```
@@ -34,46 +34,154 @@ async function handleMessage(sock: WASocket, message: WAMessage) {
+        let response = ''
+
+        switch (intent.type) {
+            case 'save_note':
+                // Guardar la nota automáticamente
+                const noteData = {
+                    titulo: intent.titulo,
+                    contenido: intent.contenido,
+                    etiqueta: intent.etiqueta
+                }
+
+                logger.info('Saving note', noteData)
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)